package.json

```diff
@@ -18,7 +18,7 @@
     "build": "cross-env-shell NODE_ENV=prod SASS_PATH=node_modules \"stencil build --config stencil.config.dist.ts\"",
     "cm": "git-cz",
     "dev": "cross-env-shell SASS_PATH=node_modules \"stencil build --dev --docs\"",
-    "watch": "cross-env-shell SASS_PATH=node_modules \"stencil build --dev --watch --docs\"",
+    "watch": "shx rm -rf .docz/ && cross-env-shell SASS_PATH=node_modules \"stencil build --dev --watch --docs\"",
```

Comment:
`shx` is the cli utility for `shelljs` (the latter of which we already use in `publish-docs.js`). This should make the `rm` command work on Windows too (although I haven't actually tested it).